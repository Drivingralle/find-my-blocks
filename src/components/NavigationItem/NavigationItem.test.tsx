import React from "react";
import renderer from "react-test-renderer";
import { NavigationItem } from "./";

describe("NavigationItem component should", () => {
  it("render correctly", () => {
    const tree = renderer.create(<NavigationItem label="label" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
