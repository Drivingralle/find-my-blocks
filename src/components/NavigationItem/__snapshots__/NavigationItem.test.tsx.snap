// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<a
  className="button"
  onClick={[Function]}
>
  <span
    className="label"
  >
    label
  </span>
  <svg
    className="feather feather-arrow-right "
    fill="none"
    height={24}
    stroke="currentColor"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="2"
    viewBox="0 0 24 24"
    width={24}
  >
    <g
      dangerouslySetInnerHTML={
        Object {
          "__html": "<line x1=\\"5\\" y1=\\"12\\" x2=\\"19\\" y2=\\"12\\"></line><polyline points=\\"12 5 19 12 12 19\\"></polyline>",
        }
      }
    />
  </svg>
</a>
`;

exports[`renders active correctly 1`] = `
<a
  className="button active"
  onClick={[Function]}
>
  <span
    className="label"
  >
    label
  </span>
  <svg
    className="feather feather-arrow-right "
    fill="none"
    height={24}
    stroke="currentColor"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="2"
    viewBox="0 0 24 24"
    width={24}
  >
    <g
      dangerouslySetInnerHTML={
        Object {
          "__html": "<line x1=\\"5\\" y1=\\"12\\" x2=\\"19\\" y2=\\"12\\"></line><polyline points=\\"12 5 19 12 12 19\\"></polyline>",
        }
      }
    />
  </svg>
</a>
`;

exports[`renders postCount correctly 1`] = `
<a
  className="button active"
  onClick={[Function]}
>
  <span
    className="label"
  >
    label
  </span>
  Found in 5 posts
  <svg
    className="feather feather-arrow-right "
    fill="none"
    height={24}
    stroke="currentColor"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="2"
    viewBox="0 0 24 24"
    width={24}
  >
    <g
      dangerouslySetInnerHTML={
        Object {
          "__html": "<line x1=\\"5\\" y1=\\"12\\" x2=\\"19\\" y2=\\"12\\"></line><polyline points=\\"12 5 19 12 12 19\\"></polyline>",
        }
      }
    />
  </svg>
</a>
`;
